docs/spec.md

```diff
@@ -2633,6 +2633,56 @@ planning to address this in
 
 &nbsp;[More Examples](https://github.com/openxla/stablehlo/tree/main/stablehlo/tests/interpret/dot_general.mlir)
 
+### dynamic_iota
+
+#### Semantics
+
+Fills a `result` tensor with values in increasing order starting from zero
+along the `iota_dimension` dimension. `result` shape is provided
+dynamically via `output_shape`.
```

Comment:
I think it would be nice to include a link to the static version of the spec here:
```
...
along the `iota_dimension` dimension. `result` shape is provided
dynamically via `output_shape`. See [`iota`](#iota).
```

Wdyt?

Reply:
agree, updated